Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -8,18 +8,28 @@
 
 #include <AzToolsFramework/API/PythonLoader.h>
 #include <AzCore/Debug/Trace.h>
+#include <AzFramework/StringFunc/StringFunc.h>
 #include <dlfcn.h>
 
 namespace AzToolsFramework::EmbeddedPython
 {
+
+
     PythonLoader::PythonLoader()
     {
-        constexpr char libPythonName[] = "libpython3.7m.so.1.0";
-        m_embeddedLibPythonHandle = dlopen(libPythonName, RTLD_NOW | RTLD_GLOBAL);
+        // PYTHON_SHARED_LIBRARY_PATH must be defined in the build scripts and referencing the path to the python shared library
+        #if !defined(PYTHON_SHARED_LIBRARY_PATH)
+        #error "PYTHON_SHARED_LIBRARY_PATH is not defined"
+        #endif
+
+        AZStd::string libPythonName;
+        // Extract only the full filename and base the library load on the python shared library that is in the library load path
+        AzFramework::StringFunc::Path::GetFullFileName(PYTHON_SHARED_LIBRARY_PATH, libPythonName);
```

Comment:
We prefer to use the AZ::IO::Path manipulation functions.

```suggestion
        // Extract only the full filename and base the library load on the python shared library that is in the library load path
                AZ::IO::FixedMaxPath libPythonName = AZ::IO::PathView(PYTHON_SHARED_LIBRARY_PATH).Filename();
```